virttest/qemu_vm.py

```diff
@@ -576,8 +576,12 @@ def add_nic(devices, vlan, model=None, mac=None, device_id=None,
                 dev.set_param('macaddr', mac, 'NEED_QUOTE', True)
             dev.set_param('id', device_id, 'NEED_QUOTE')
             if "virtio" in model:
-                if int(queues) > 1:
+                if mq is None and int(queues) > 1:
                     dev.set_param('mq', 'on')
+                elif mq and int(queues) > 1:
+                    dev.set_param('mq', mq)
```

Comment:
```
if int(queues) > 1:
    mq = 'on' if mq is None else mq
    dev.set_param('mq', mq)

Reply:
I have updated,please help review it again.Thanks a lot.